src/regmod/utils.py

```diff
@@ -40,11 +114,31 @@ def __post_init__(self):
 
     @property
     def num_spline_bases(self) -> int:
+        """Number of the spline bases.
+        """
```

Comment:
If your doc string is one line, put the ending quotes on the same line:
`"""Return number of the spline bases."""`

Reply:
Yes! right on! Changed to one line, and here is the reference.
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings